company.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+    <div class="spinner-border text-secondary d-none" style="width: 2.5rem; height: 2.5rem;" id="spinnerForCompany" role="status">
+      </div>
+
+    <div id="companyInfoContainer" class="companyInfoContainer">
+      <div class="name-and-link" id="nameNlink"></div>
+      <div id="compImageNprice" class="company-image">
+      <div id="compImage"></div>
+      <div id="compPrice"></div>
+      <div id="priceChange"></div>
+      </div>
+      <div class="description" id="compDescription"></div>
+      <div class="charts">
+        <canvas id="myChart"></canvas>
+      </div>
+    </div>
+
+    <script src="/js/companyInfo.js"></script>
```

Comment:
this should be the syntax for using relative path files 
 src="./js/companyInfo.js"